.github/workflows/post-release-update_bpmn_visualization_version_in_Examples_repo.yml

```diff
@@ -1,4 +1,4 @@
-name: Update BPMN Visualization version in Examples repo
+name: Update the version of the bpmn-visualization TypeScript library in Examples repo
```

Comment:
match changes in https://github.com/process-analytics/bpmn-visualization-examples/pull/436